src/routes/Project/actions/ClipActionHandlers.js

```diff
@@ -0,0 +1,13 @@
+import ProjectConstants from '../constants';
+
+const ACTION_HANDLERS = {
+  [ProjectConstants.ADD_CLIP]: (state, action) => {
+    const id = action.payload.id;
+    const track = action.payload.start.y;
+    const startTime = action.payload.start.x;
+    const endTime = action.payload.end.x;
+    return { ...state,  [id]: { track, startTime, endTime } };
```

Comment:
Ya its the map of clips for the project, I just realized its missing audio attribute which will store a path to the audio file, most likely an s3 link